Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM node
-
-COPY package*.json ./
+FROM node:12-slim as build
+WORKDIR /usr/secret-backup-operator
+COPY ./package*.json ./
 RUN npm ci
+COPY . .
 
-COPY . . 
-
-CMD ["npm", "start"]
\ No newline at end of file
+FROM build as test
+CMD ["npm", "run", "test"]
```

Comment:
Currently the published images default command is test, which is not good i think.
I would add an other stage thats cmd is npm start and publish that to the registry.